spec/uploader/processing_spec.rb

```diff
@@ -52,10 +52,10 @@
 
     context "when there is an 'if' condition" do
       it "calls the processor if the condition method returns true" do
-        uploader_class.process :resize => [200, 300], :if => :true?
+        uploader_class.process :resize => [200, 300, combine_options: { quality: 70 }], :if => :true?
         uploader_class.process :fancy, :if => :true?
         expect(uploader).to receive(:true?).with("test.jpg").twice.and_return(true)
-        expect(uploader).to receive(:resize).with(200, 300)
+        expect(uploader).to receive(:resize).with(200, 300, combine_options: { quality: 70 })
```

Comment:
Do not modify an unrelated spec, you should create a new one instead.

Reply:
Sure! thanks for the review!